README.md

```diff
@@ -80,26 +80,26 @@ All of the templates are written in [Handlebars.js](https://handlebarsjs.com/),
 
 Highlighted items from src/js/state.js for use in templates.  See src/js/state.js for more.
 
-- `form.serverName` - Server Name
-- `form.serverVersion` - Server Version
-- `form.opensslVersion` - OpenSSL Version
+- `form.serverName` - display name of the server
+- `form.serverVersion` - desired server version
+- `form.opensslVersion` - desired OpenSSL version
```

Comment:
"desired"?  I am not sure that describes "generate config to work with this version" (if version is recognized)

Reply:
Yes please do challenge any awkward language you come across. Thanks!

I wanted to make a clear difference between options read from configs as compared to values entered by the visitor through the UI at the time of every render.

Do you have any idea for a reasonably brief wording? Something equivalent to "… version requested by the user" perhaps?

PS: "config to work with this version" is not exactly achieved yet, as the value entered might be artificially high or unreasonably low… but it's the value passed from the user nonetheless.

Reply:
"user's server version"?  "your server version"?  "input server version"? "user-selected server version"? "user-specified server version"?  I lean towards one of the last two.

Reply:
I think I'll go with "requested", as we don't know if it's actually "entered", "selected" or "specified" by the user as their version at all, as it may just come from the default value being pre-populated by configs settings on load; guess I'll be as neutral as possible.